Formula/k/ktor.rb

```diff
@@ -0,0 +1,26 @@
+class Ktor < Formula
+  desc "Generates Ktor projects through the command-line interface"
+  homepage "https://github.com/ktorio/ktor-cli"
+  url "https://github.com/ktorio/ktor-cli.git",
+      tag:      "0.2.0",
+      revision: "31bd486d11defbb573433dbb9f79f6e3e7ca407b"
+  license "Apache-2.0"
+  head "https://github.com/ktorio/ktor-cli.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "0"
+    system "go", "build",
+           *std_go_args(ldflags: "-X main.Version=#{stable.specs[:tag]}"), "github.com/ktorio/ktor-cli/cmd/ktor"
+  end
+
+  test do
+    system bin/"ktor", "new", "project"
+    assert_predicate Pathname.new("project/build.gradle.kts"), :exist?
```

Comment:
```suggestion
    assert_path_exists testpath/"project/build.gradle.kts"
```
and similarly below.